scripts/go.mod

```diff
@@ -278,3 +278,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/spf13/viper => github.com/spf13/viper v1.12.0
```

Comment:
why do we need this replace?

Reply:
I'd rather downgrade golangci-lint or wait for the upstream fix.

Reply:
Thanks @nicolastakashi @simonpasquier 
Golangci-lint use `spf13/viper` v1.12.0 version. ref: https://github.com/golangci/golangci-lint/blob/master/go.mod#L106